charts/fulcio/Chart.yaml

```diff
@@ -5,8 +5,8 @@ description: |
 
 type: application
 
-version: 0.5.2
-appVersion: 0.6.0
+version: 0.9.9
```

Comment:
I had a look and I believe we could create a Chart with `version: 1.0.0-rc.0`, cc @cpanato

Reply:
Updated!